### PR TITLE
[harbour-validator] Allow WebView API. Fixes JB#55425

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: allowed_requires.conf [hunk 1]
```diff
@@ -27,6 +27,10 @@ libpulse-simple.so.0(*)
 sailfishsilica-qt5
 libsailfishapp
 mapplauncherd-booster-silica-qt5
+libqt5embedwidget.so.1(*)
+sailfish-components-webview-qt5
```
Comment (rainemak) [inline allowed_requires.conf:31]: And this make plugin libs as allowed?

Comment (vigejolla) [reply allowed_requires.conf:31]: I don't think you are supposed to link against the plugin libs. When you import Sailfish.WebView 1.0 the qml engine loads the needed plugin. But now I'm wondering, should we add also "Sailfish.WebEngine", "Sailfish.WebView.Controls", "Sailfish.WebView.Pickers" and "Sailfish.WebView.Popups" to the list of allowed qml imports?

Comment (rainemak) [reply allowed_requires.conf:31]: Yes, all them should be there in the allowed list.

Comment (rainemak) [reply allowed_requires.conf:31]: Some API are documented as "can change whenever" but those have been the same for let's say +5 years.

Comment (vigejolla) [reply allowed_requires.conf:31]: added those

File: allowed_requires.conf [hunk 2]
```diff
@@ -27,6 +27,10 @@ libpulse-simple.so.0(*)
...
+sailfish-components-webview-qt5-popups
```
Comment (rainemak) [inline allowed_requires.conf:32]: This as well:
sailfish-components-webview-qt5-pickers

Comment (vigejolla) [reply allowed_requires.conf:32]: added

**Review by rainemak — APPROVED** [review]

**Pull request merged**